lib/models/os.coffee

```diff
@@ -54,8 +54,10 @@ getOsModel = (deps, opts) ->
 			{ versions, latest } = body
 			[ validVersions, invalidVersions ] = partition(versions, semver.valid)
 
-			validVersions.sort(osVersionRCompare)
```

Comment:
You're right - I'll fix this, was just trying to avoid the sort thinking we didn't need it